projects/observability/src/shared/components/entity-renderer/entity-renderer.component.ts

```diff
@@ -15,7 +15,7 @@ import { EntityNavigationService } from '../../services/navigation/entity/entity
       class="ht-entity-renderer"
       [ngClass]="{ navigable: this.navigable }"
       [htTooltip]="this.name"
-      (click)="this.onClickNavigate()"
+      (click)="this.navigable && this.onClickNavigate()"
```

Comment:
Should this check be put inside?

Reply:
Eh, its a fair ask but not sure its really necessary here. Just more lines of code and I don't feel it is any cleaner. If you feel strongly I can change though.

Reply:
cool